core/graph/src/test/java/uk/gov/gchq/gaffer/graph/hook/AdditionalOperationsTest.java

```diff
@@ -105,6 +109,7 @@ public void assertClonedOperations(final List<Operation> ops1, final List<Operat
 
     @Override
     protected AdditionalOperations getTestObject() {
-        return new AdditionalOperations();
+        return fromJson(ADDITIONAL_OPERATIONS_RESOURCE_PATH);
```

Comment:
These tests objects should normally just be a new instance of the class, in the background for some things it does some checks for required fields etc which if they are populated will not throw exceptions correctly.  Can you change this back to just the ```new AdditionalOperations();```